### DISCUSSION TITLE
v1.15.0 lotus-miner sealing is not same as config

### DISCUSSION THREAD

**Discussion by jzmexec** [filecoin-project/lotus#8350] [General Q&A] [answered]
I set the lotus-miner config as "CommittedCapacitySectorLifetime = "4320h0m0s"".
But, I get the cc sectors is "2253571 (in 30 weeks 16 hours)".

Can you help me, and how to set the CommittedCapacitySectorLifetime config?

**Comment by jzmexec** [comment]
<img width="507" alt="1647711431(1)" src="https://user-images.githubusercontent.com/71239926/159132073-a11d58ff-37d1-4278-8ee9-6de5255387b3.png">

**Reply by stuberman** [reply]
4320 hours == 180 days

In your .lotusminer/config.toml set/add this variable and then restart the lotus-miner.
You can only set the duration of new CC sectors before you pledge that sector.
You cannot modify existing sectors

> [Sealing]
>   CommittedCapacitySectorLifetime = "4320h0m0s"

**Reply by stuberman** [reply]
FYI - There is a bug in the code such that you cannot actually set the minimum of 180 days (25.7 weeks), but a bit more than 200 days (30 weeks)

**Comment by jzmexec** [comment]
@stuberman thanks, I'm sure i restart lotus-miner and it is new CC sectors.

**Comment by rjan90** [comment, marked as answer]
As @stuberman commented above, there is a bug in the code where it takes into account the `MaxProveCommitDuration` when creating a CC-sector, which does not make it possible to create a 180 day CC-sector, only approx 210 days as minimum. The issue is being tracked here: https://github.com/filecoin-project/lotus/issues/7609
